README.md

```diff
@@ -28,8 +28,10 @@ To get a bot to join a Google Meet, send a POST request to the join endpoint wit
 curl --header "Content-Type: application/json" \
   --request POST \
   --data '{"url":"https://meet.google.com/wtq-bhai-amg"}' \
-  http://localhost:8080/join
+  http://localhost:8080/api/join
 ```
+## UI Development
+To develop the UI, `cd` into `/src/ui` and run `yarn/npm run dev`. This will spin up the development server. You should then be able to access the dashboard at `http://localhost:3000/ui`. Note that the data you will see on the tables are actually being mocked by MirageJS, and not actual request is going to the API. This makes it easier to develop both components in an entirely decoupled way.
```

Comment:
I thought all we gotta do is change this since the ui can be served from the root of the project now. Also, it serves on port 5000 with the serve:ui command so mention that unless the dev version is port 3000 (probably make them the same to avoid confusion.

Reply:
You're right. It's just the README that was outdated. Should be ok now. The port of the preview server was too updated according to what you suggested.